website/docs/components/card/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Card
+description: A block container that provides styling for elevation, border and background.
```

Comment:
Add oxford comma:
"A block container that provides styling for elevation, border and background."
=>
"A block container that provides styling for elevation, border, and background."